kg_microbe/transform_utils/bacdive/bacdive.py

```diff
@@ -1042,18 +1048,85 @@ def run(self, data_file: Union[Optional[Path], Optional[str]] = None, show_statu
+                                    ncbi_label = get_label(self.ncbi_impl, ncbitaxon_id)
+                                    break
+
+                        if ncbitaxon_id is None:
+                            full_name = name_tax_classification.get(FULL_SCIENTIFIC_NAME, "unknown")
+                            print(f"WARNING: BacDive {key} - no NCBITaxon match found for: {full_name}")
+
+                    # Create strain node for every BacDive record
+                    # Extract strain designation and TYPE_STRAIN status
+                    strain_designation = None
+                    type_strain = None
+                    if name_tax_classification:
+                        if name_tax_classification.get(STRAIN_DESIGNATION):
+                            strain_designation = name_tax_classification.get(STRAIN_DESIGNATION).strip()
+                        type_strain = name_tax_classification.get(TYPE_STRAIN)
+
+                    # Construct strain ID using BacDive ID
+                    organism_id = STRAIN_PREFIX + BACDIVE_PREFIX.replace(":", "_") + key
+
+                    # Construct strain label
+                    bacdive_key = BACDIVE_PREFIX.replace(':', '_') + key
+                    if strain_designation and ncbitaxon_id:
+                        organism_label = f"{bacdive_key} as {strain_designation} of {ncbitaxon_id}"
+                    elif ncbitaxon_id:
+                        organism_label = f"{bacdive_key} strain of {ncbitaxon_id}"
+                    elif name_tax_classification and name_tax_classification.get(FULL_SCIENTIFIC_NAME):
+                        scientific_name = name_tax_classification.get(FULL_SCIENTIFIC_NAME)
+                        organism_label = f"{bacdive_key} strain of {scientific_name}"
+                    else:
+                        organism_label = f"{bacdive_key} strain"
+
+                    # Write strain node
+                    node_writer.writerow(
+                        [
+                            organism_id,
+                            NCBI_CATEGORY,
+                            organism_label,
+                        ]
+                        + [None] * (len(self.node_header) - 3)
+                    )
+
+                    # Write subClassOf edge to NCBITaxon (if available)
+                    if ncbitaxon_id:
+                        edge_writer.writerow(
+                            [
+                                organism_id,
+                                SUBCLASS_PREDICATE,
+                                ncbitaxon_id,
+                                RDFS_SUBCLASS_OF,
+                                BACDIVE_PREFIX + key,
+                            ]
+                        )
+
+                    # Always use only the strain node for feature edges
+                    # NCBITaxon relationship is preserved via subclass edge (strain -> NCBITaxon)
+                    feature_targets = [organism_id]
```

Comment:
If this is just going to be organism_id why do we need to convert it to a list? We can simply use it as is and revert the change in logic in methods like `_process_phenotype_by_metpo_parent()`, `_process_antibiotic_resistance()`, `_process_metabolites()`, etc.